### ISSUE TITLE
Spacing in curly brackets

### ISSUE THREAD

**Issue by hirokiosame** [vuejs/eslint-plugin-vue#150]
**Please describe what the rule should do:**
Enforce spacing on the style of curly brackets

**What category of rule is this? (place an "X" next to just one item)**

[x] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```vue
<template>
     <div>
          {{text}}
     </div>
</template>
```

```vue
<template>
     <div>
          {{ text }}
     </div>
</template>
```

**Comment by mysticatea** [comment]
Thank you for this proposal!

Sounds good to me.

**Comment by armano2** [comment]
I think we should consider adding 2 options: `always` and `never`
```
curly-bracket-spacing: ['error', 'always|never']
```
